tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption("--runslow", action="store_true", default=False, help="run slow tests")
+
+
+def pytest_configure(config):
```

Comment:
What is all this doing? Isn't there a native option to mark tests as slow that we've already been using?

Reply:
Not that I'm aware of.

Reply:
It does look like there's a package we can download if we'd like to instead: https://pypi.org/project/pytest-skip-slow/ 

I'm surprised this isn't just default pytest